code/modules/research/designs/engineering.dm

```diff
@@ -31,6 +31,14 @@
 	build_path = /obj/item/weapon/tool/crowbar/power
 	sort_string = "NAAAC"
 
+/datum/design/item/tool/rpd
```

Comment:
```suggestion
/datum/design/item/tool/rpd // YW addition begins
```

Reply:
@RadiantFlash7 @izac112 *Or* someone can suggest changes and have Radiant accept them. \:3